apps/admin-api/env/.env.unit.test

```
@@ -0,0 +1,12 @@
+POSTGRES_DB_URI=postgres://postgres:password@localhost:5432/console-users
+PORT=3010
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PORT key should go before the POSTGRES_DB_URI key

(UnorderedKey)